package.json

```diff
@@ -66,7 +67,9 @@
     "d3-array": "2.8.0",
     "d3-timer": "2.0.0",
     "istanbul-instrumenter-loader": "3.0.1",
+    "lodash": "^4.17.21",
```

Comment:
Do not load the full `lodash` library here. This library always has problems with vulnerabilities.
Instead, import separate packages like: https://www.npmjs.com/package/lodash.flatmap